### DISCUSSION TITLE
Are NV cards recommended over AMD cards regarding latency?

### DISCUSSION THREAD

**Discussion by shanhe-bc3** [LizardByte/community#256] [Sunshine]
Try to decide which card to buy for mostly remote playing.

Have read about NV has capture mode like nvfbc seems to be better than AMD cards on latency.

Also if use AMD card, is it possible to encode using QuickSync for lower latency?

**Comment by ReenigneArcher** [comment]
Quick Sync is an Intel technology, not AMD.

As for which cards are better I don't provide recommendations. Perhaps some community members can share their experiences if they've tried both brands.

**Reply by shanhe-bc3** [reply]
I mean like games run on AMD, but use intel igpu to encode.

**Reply by ReenigneArcher** [reply]
Sunshine doesn't care what GPU for your games, it cares what GPU is rendering the display.

[a comment by an automated account is not shown]
